apps/avifenc.c

```diff
@@ -120,7 +120,7 @@ static void syntax(void)
         printf("    cq-level=Q                        : Constant/Constrained Quality level (0-63, end-usage must be set to cq or q)\n");
         printf("    enable-chroma-deltaq=B            : Enable delta quantization in chroma planes (0: disable (default), 1: enable)\n");
         printf("    end-usage=MODE                    : Rate control mode (vbr, cbr, cq, or q)\n");
-        printf("    sharpness=S                       : Loop filter sharpness (0-7, default: 0)\n");
+        printf("    sharpness=S                       : Bias towards block sharpness in transform coeff RD (0-7, default: 0)\n");
```

Comment:
Since this is documentation for end users, we should avoid technical abbreviations. Could you please change "transform coeff RD" to "transform coefficient rate-distortion optimization" or something like that? Thanks.

Reply:
Hi @wantehchang
We have updated the documentation as per your comments.
Please review.